src/configure/configure.ts

```diff
@@ -154,6 +157,23 @@ class Orchestrator {
         }
     }
 
+    private async getGithubPatToken(): Promise<void> {
+        const redirectHelper = new RedirectLinkHelper();
+        await redirectHelper.loadAll();
```

Comment:
you are calling this at multiple places -  then loadAll seems like a wrong pattern. LoadAll signifies that it should be loaded just once. Either load it really just once or directly expose a function called getRepositoryAnalysisUrl and implement a caching as suggested in other comment